src/proxies_v8/proxies_v8.ts

```diff
@@ -1100,3 +1100,8 @@ proxiesV8.use('/national/learning/week/insights',
   // tslint:disable-next-line: max-line-length
   proxyCreatorSunbirdSearch(express.Router(), `${CONSTANTS.KONG_API_BASE}/national/learning/week/insights`)
 )
+
```

Comment:
No need to write this 
https://github.com/KB-iGOT/sunbird-cb-uiproxy/blob/466a6a4e802805e6846d54a0b8822e17afd60fec/src/proxies_v8/proxies_v8.ts#L993C1-L995C2